src/Compilers/CSharp/Portable/Symbols/SymbolWithAnnotations.cs

```diff
@@ -267,7 +371,7 @@ public string ToDisplayString(SymbolDisplayFormat format = null)
             return str;
         }
 
-        internal string GetDebuggerDisplay() => ToDisplayString(DebuggerDisplayFormat);
+        internal string GetDebuggerDisplay() => _defaultType is null ? "null" : ToDisplayString(DebuggerDisplayFormat);
```

Comment:
>GetDebuggerDisplay [](start = 24, length = 18)

nit: would it be useful to display something about the extension too?

Reply:
Let's add that later, when it's more obvious what to display for `_extension`.

---
In reply to: [206655276](https://github.com/dotnet/roslyn/pull/28943#discussion_r206655276) [](ancestors = 206655276)